### DISCUSSION TITLE
Change default Fan Speed

### DISCUSSION THREAD

**Discussion by guinnberg** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#353] [Q&A] [answered]
Every time the vacuum finishes to clean, the fan speed goes back to _standard_. How can I make it always to be a default one? I usually just use _automatic_

**Comment by PiotrMachowski** [comment, marked as answer]
The best solution is to create an automation that will do it

**Reply by guinnberg** [reply]
Ok thanks.
I understand then it's the vacuum resetting this attribute itself?

**Reply by PiotrMachowski** [reply]
Yup, it's not done by the card
